Source/JavaScriptCore/assembler/MacroAssemblerARM64.h

```diff
@@ -4979,6 +4979,11 @@ class MacroAssemblerARM64 : public AbstractMacroAssembler<Assembler> {
         m_assembler.vectorEor(dest, left, right);
     }
 
+    void vectorMoveZero(FPRegisterID dest)
+    {
+        vectorXor({ SIMDLane::v128, SIMDSignMode::None }, dest, dest, dest);
+    }
```

Comment:
Probably we should name it `moveZeroToVector`, aligned to `moveZeroToDouble`, `moveVector`, `moveDouble`.